source/command-logging-and-monitoring/command-logging-and-monitoring.rst

```diff
@@ -341,42 +342,217 @@ See the `Load Balancer Specification <../load-balancers/load-balancers.rst#event
   }
 
+------------
+Log Messages
+------------
+Please refer to the `logging specification <../logging/logging.rst>`_ for details on logging implementations in general, including log levels, log
+components, and structured versus unstructured logging.
 
---------
-Examples
---------
+Drivers MUST support logging of command information via the following types of log messages. These messages MUST be logged at ``Debug`` level and use
+the ``command`` log component.
+
+The log messages are intended to match the information contained in the events above. Drivers MAY implement command logging
+support via an event subscriber if it is convenient to do so.
+
+The types used in the structured message definitions below are demonstrative, and drivers MAY use similar types instead so long as the information
+is present (e.g. a double instead of an integer, or a string instead of an integer if the structured logging framework does not support numeric types.) 
+
+For the string representations of messages defined below, for any optional field that is not available, drivers MAY either substitute the value with a
+language-appropriate representation of null (e.g. "null", "nil", "None") or omit the corresponding segment of the message altogether. 
+
+Drivers MUST not emit command log messages for commands issued as part of the handshake with the server, or heartbeat commands issued by server monitors. 
+
+Command Started Message
+-----------------------
+The structured format MUST contain the following key-value pairs:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 1 1 1
+
+   * - Key
+     - Suggested Type
+     - Value
+
+   * - message
+     - String
+     - "Command started"
+
+   * - command
+     - String
+     - Canonical extJSON representation of the command. This document MUST be truncated appropriately according to rules defined in the 
+       `logging specification <../logging/logging.rst>`_, and MUST be replaced with an empty document "{ }" if the command is considered sensitive.
+
+   * - databaseName
+     - String
+     - The database name.
+
+   * - commandName
+     - String
+     - The command name.
+
+   * - requestId
+     - Int
+     - The driver-generated request ID.
+  
+   * - operationId
+     - Int
+     - The driver-generated operation ID. Optional; only present if the driver generated operation IDs and this command has one. 
+
+   * - driverConnectionId
+     - Int
+     - The driver's ID for the connection used for the command, as defined in the `connection monitoring and pooling specification <../connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst>`_. 
+
+   * - serverHost
+     - String
+     - The hostname or IP address for the server the command is being run on.
+
+   * - serverPort
+     - Int
+     - The port for the server the command is being run on. Optional; only present if a port was specified.
 
-A Ruby subscriber to a query series and how it could handle it with respect to logging.
```

Comment:
this example seemed like it would be confusing in light of actual logging info being added to the spec. I didn't bother to try to write a replacement example but I can do so if anyone thinks that is needed.